Aptoide/aptoide-client-v8
# [MOB-460] Delete Realm

**What does this PR do?**

   This PR aims at removing realm from the project after we implemented room. The only database table left with realm was the Store table as we are saving the stores for the non logged in users o that table.

**Database changed?**

   Yes 

**Where should the reviewer start?**

- [ ] Store.java

**How should this be manually tested?**

Run dev, then run this branch. Check that nothing breaks. Also do tests around the app with things that use the database, like downloads, updates ,etc.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-460](https://aptoide.atlassian.net/browse/MOB-460)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
